lib/internal/test_runner/test.js

```diff
@@ -1002,6 +1003,21 @@ class Test extends AsyncResource {
 
       if (coverage) {
         reporter.coverage(nesting, loc, coverage);
+
+        if (coverage.totals.coveredLinePercent < harness.minLineCoverage) {
+          process.exitCode = 1;
```

Comment:
Please use `kGenericUserError` like other places in the test runner. Same comment for branches and functions below.